sdk/profiles/hybrid20200901/definition.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
This info looks redundant with what's in the autorest.md files.  Can it be removed?

Reply:
This config is used to generate autorest.md files for all the profile packages.